### PR TITLE
Update free-courses-ne.md

### PR DESCRIPTION
**Pull request by AryavT**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Class objective variable in java for Nepalis language
### Why is this valuable (or not)?
this is valuable because it will be availed to the Nepalis community that I relate to and we will be able to succeed and be seen.

### How do we know it's really free?
it is a YouTube video with now fees
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes
https://www.youtube.com/playlist?list=PLmZYUigljiyc-tf7oMmM-s832ibhGxTpT

**Comment by AryavT** [comment]
> For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes https://www.youtube.com/playlist?list=PLmZYUigljiyc-tf7oMmM-s832ibhGxTpT

finished

File: courses/free-courses-ne.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 ### JavaScript
 
 * [JavaScript](https://www.youtube.com/playlist?list=PLckS_N3kOwFH-GCqCd6i-vPo-Z75DcOnc) - EverydayKarma 🇳🇵
+* [JAVA PROGRAMMING - TUTORIAL - NEPALI](https://www.youtube.com/playlist?list=PLmZYUigljiyc-tf7oMmM-s832ibhGxTpT) - BigData IT
```
Comment (eshellman) [inline courses/free-courses-ne.md:16]: no ALL CAPS

File: courses/free-courses-ne.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
 ### JavaScript
 
 * [JavaScript](https://www.youtube.com/playlist?list=PLckS_N3kOwFH-GCqCd6i-vPo-Z75DcOnc) - EverydayKarma 🇳🇵
+* [Java Programming - Tutorial - Nepali](https://www.youtube.com/playlist?list=PLmZYUigljiyc-tf7oMmM-s832ibhGxTpT) - BigData IT
```
Comment (eshellman) [inline courses/free-courses-ne.md:16]: Alphabetization: move `java P` before `javasc`

**Pull request merged**